index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Integer to Roman Numeral Conversion</title>
+  </head>
+  <body>
+    <label for="integer">Enter an integer between 0 and 3,999: </label>
+    <input id="integer" type="number" min="1" max="3999">
```

Comment:
The min/max here stop the user from increasing the number past 3999, BUT only when they use the up/down arrows. 
If you type in a larger number it bypasses that (annoyingly). Like 4000000 puts out a lot of M's. 
I'd add a check/block in the JS code too